src/context/FilterProjectsContext.jsx

```diff
@@ -187,8 +198,19 @@ export const FilterProjectsProvider = ({ children }) => {
     showYourData,
     methodDataSharingFilters,
     projectNameFilter,
+    enableFollowScreen,
   ])
 
+  const normalizeLng = (lng) => {
+    if (lng < -180) {
+      return lng + 360
+    }
+    if (lng > 180) {
+      return lng - 360
+    }
+    return lng
+  }
+
```

Comment:
Have you considered what would happen we'd crossed the date line twice? e.g. < -540 or > 540?
A few sources online suggest using the modulo operator (`%`) to solve this.